packages/dnb-eufemia/src/extensions/forms/Iterate/Toolbar/ToolbarDocs.ts

```diff
@@ -1,5 +1,9 @@
 import { PropertiesTableProps } from '../../../../shared/types'
 
-export const ToolbarProperties: PropertiesTableProps = {}
-
-export const ToolbarEvents: PropertiesTableProps = {}
+export const ToolbarProperties: PropertiesTableProps = {
+  variant: {
+    doc: 'Use variants to render the toolbar differently. See the info section for more info.',
```

Comment:
```suggestion
    doc: 'Use variants to render the toolbar differently. See the info section for more info.',
```

Which values of variant is available?